src/html/scope.rs

```diff
@@ -108,20 +108,26 @@ impl<COMP: Component> Scope<COMP> {
     }
 
     /// Send a message to the component
-    pub fn send_message(&self, msg: COMP::Message) {
-        self.update(ComponentUpdate::Message(msg));
+    pub fn send_message<T>(&self, msg: T)
+    where
+        T: Into<COMP::Message>,
+    {
+        self.update(ComponentUpdate::Message(msg.into()));
     }
-
     /// Send a batch of messages to the component
-    pub fn send_message_batch(&self, messages: Vec<COMP::Message>) {
-        self.update(ComponentUpdate::MessageBatch(messages));
+    pub fn send_message_batch<I: Into<Vec<COMP::Message>>>(&self, messages: I)
```

Comment:
Looks like you have two trait bounds on this method. `I: Into<Vec<COMP::Message>>` and `I: IntoIterator<Item = COMP::Message>`. We only need `I: IntoIterator<Item = COMP::Message>`. Then, you can use the `into_iter()` method to change `messages` input into a vec of messages

Reply:
I've tried to use only IntoIterator, but I'm receiving `expected struct `std::vec::Vec`, found associated type`.
So, I don't know if I can change implementation of `MessageBatch`

Reply:
did you try this?
```rust
messages.into_iter().collect()
```
If you have an `IntoIterator` you need to first turn it into an iterator, and then you can convert it into a vec to pass to `MessageBatch`

Reply:
It works, thank you.